### ISSUE TITLE
Missing primary keys from a few tables

### ISSUE THREAD

**Issue by bjagg** [Jasig/uPortal#2162]
**Describe the bug**
There are missing primary keys in 3 tables. The cause seems to be due to using `@jointable` rather than `@collectiontable`

	- UP_EVENT_AGGR_CONF_INTRVL_EXC
	- UP_EVENT_AGGR_CONF_INTRVL_INC
	- UP_SS_DESC_LAY_ATTR_ELMS

**To Reproduce**
1. run `gradlew dataInit`

**Expected behavior**
All uPortal tables have primary keys defined

**Comment by bjagg** [comment]
https://www.logicbig.com/tutorials/java-ee-tutorial/jpa/collection-table.html

https://github.com/Jasig/uPortal/blob/271f309ed71d57396737ef0c56f2a2dff4b61a73/uPortal-layout/uPortal-layout-core/src/main/java/org/apereo/portal/layout/dao/jpa/LayoutAttributeDescriptorImpl.java

**Comment by jgribonvald** [comment]
Good catch ! The thing is that the ID column exist but they aren't defined as PrimaryKeys ! 
The patch is easy, like the migration for existing datas.
